common/lib/xmodule/xmodule/modulestore/inheritance.py

```diff
@@ -100,6 +100,26 @@ class InheritanceMixin(XBlockMixin):
         scope=Scope.settings,
         default="finished",
     )
+
+    show_correctness = String(
```

Comment:
@pomegranited I didn't realise that it now contains changes to CAPA module as well.

So, I've got some controversial feelings about this. On one hand- it's a very elegant solution - and a richer one than I pictured. If I understand right, this now allows CAPA problems to override show_correctness if needed, but if not they take it form subsection.

On the other hand, I've got some concerns that edX might push back on this, because essentially it adds show_correctness setting to every module using inheritance mixin, which is a bit broader than what we actually need.

So, I'd suggest to keep it and wai tand see if edX reviewers welcome this or push it back.